README.md

```diff
@@ -0,0 +1,48 @@
+# Analytics
+
+Interface web para consulta de indicadores de acessos e publicação das coleções da rede SciELO.
+
+
+### Arquivos: Dockerfile* e docker-compose*.yml
+
+
+Dockerfile:
+- **Dockerfile**: contém as definições para construir a imagem pronta para instalar em **produção**
+- **Dockerfile-dev**: contém as definições para construir a imagem pronta para instalar em **desenvolvimento**
+
+Docker Compose:
+- **docker-compose.yml**: contém as definições para iniciar todos os containers necessários para rodar em **produção**
+- **docker-compose-dev.yml**: contém as definições para iniciar todos os containers necessários para rodar em **desenvolvimento**
+
+
+### Como executar os tests
+
+
+
+- Para rodar os tests de unidade, pode executar: ``python setup.py test``
+- Para ter o relatório de coverage deve executar: ``python setup.py converage``
```

Comment:
Seria `coverage` ao invés de `converage`?

Reply:
Fazendo `docker exec -it analytics_webapp_1 python setup.py test`  ocorre `error: invalid command 'coverage'`. Instalei via `docker-compose -f docker-compose-dev.yml build`.

Reply:
Removi esse comentário do README.md para que em outro momento possamos documentar.